test/integration/array_test.ts

```diff
@@ -626,4 +629,43 @@ describe('Array', function () {
       assert.equal('{"k1":[7,1,5,3,6,4]}', d1.toJSON());
     }, this.test!.title);
   });
+
+  it('Can get deleted element after delete operation', function () {
+    const doc = DocumentReplica.create('test-col', 'test-doc');
+    let prevID: TimeTicket;
+
+    doc.update((root) => {
+      root['list'] = [0, 1, 2];
+      prevID = root['list'].getElementByIndex(1).getID();
+      assert.equal('{"list":[0,1,2]}', root.toJSON());
+    });
+
+    doc.update((root) => {
+      root['list'].deleteByID(prevID);
+      const elem = root['list'].getElementByID(prevID);
+      assert.equal(1, elem);
+    });
+  });
```

Comment:
I wonder if this test behavior is correct behavior.🤔

Reply:
We missed this situation. It would be good to register as a new issue.